Code/Framework/AzFramework/AzFramework/Physics/CharacterBus.h

```diff
@@ -103,4 +104,75 @@ namespace Physics
     };
 
     using CharacterRequestBus = AZ::EBus<CharacterRequests>;
-} // namespace Physics
+
+    /// Messages sent by character controllers.
+    class CharacterNotifications : public AZ::ComponentBus
+    {
+    private:
+        template<class Bus>
+        struct CharacterNotificationsConnectionPolicy : public AZ::EBusConnectionPolicy<Bus>
```

Comment:
question: Based on the recent EBus changes by @lawsonamzn I wonder if it's preemptively worth moving this to the .cp and using the new `extern` approach? It might be worth waiting to profile the compile time I just wanted to sanity check in case.

Reply:
I believe @lawsonamzn specifically said _not_ to preemptively use the extern approach, that it should only be used for highly-used EBuses? This one seems like it would be more sparsely used IMO...